languages/go/exercises/concept/time-duration/.docs/after.md

```diff
@@ -0,0 +1,14 @@
+A [`Time`][time] in Go is a type describing a moment in time. The date and time information can be accessed, compared, and manipulated through its methods, we have read earlier about some functions called on the `time` package itself. In this exercise we are going to learn about [`Duration`][duration] package.
```

Comment:
As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd), the after.md document should contain all information that is introduced in the introduction.md document. This does not seem to be the case here, as the parsing remark e.g. is not included in this document.